src/alto-ui/Dialog/__tests__/Dialog.test.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Dialog from '../Dialog';
+
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return props => {
+    return <button className="CloseButton" onClick={() => props.onClick()} />;
+  };
+});
+
+describe('Dialog', () => {
+
+  const defaultProps = {
+    id: 'string',
+    // className: 'string',
+    // open: true,
+    // title: 'string',
+    // onClose: jest.fn(),
+    // buttons: [{}, {}],
+    // closeFocusTargetId: 'string',
+    // inert: true,
+    // a11yLabelClose: 'string',
+  }
+
+  const getWrapper = ({ ...props }) => shallow(<Dialog {...defaultProps} {...props} />);
+
+  it('is Dialog',() => {
+    const wrapper = getWrapper({
+      id: 'key'
+    });
+    expect(wrapper.find('.Dialog__overlay').exists()).toBe(true);
+  })
+
+  it('should render div with Dialog className',() => {
+    const props = {
+      open: true,
+    }
+
+    const wrapper = getWrapper(props);
+    expect(wrapper.find('.Dialog').exists()).toBe(true);
+  })
+
+  it('should render header with Dialog__header className',() => {
+    const propsTitle = {
+      open: true,
+      title: 'string'
+    }
+
+    const propsClose = {
+      open: true,
+      onClose: jest.fn()
+    }
+
+    expect(getWrapper(propsTitle).find('.Dialog__header').exists()).toBe(true);
+    expect(getWrapper(propsClose).find('.Dialog__header').exists()).toBe(true);
+  })
+
+  it('should render h2 with Dialog__title className',() => {
```

Comment:
should has and write negative test too

Reply:
I wrote negative tests:
`open: false`  =>  `it('should has Overlay element')`
`title: '', onClose: undefined`  => `it('should no has element with Dialog__header className')`